paramak/parametric_neutronics/neutronics_model.py

```diff
@@ -175,6 +179,28 @@ def mesh_tally_2D(self, value):
                         output_options)
         self._mesh_tally_2D = value
 
+    @property
+    def mesh_tally_3D(self):
+        return self._mesh_tally_3D
+
+    @mesh_tally_3D.setter
+    def mesh_tally_3D(self, value):
+        if value is not None:
+            if not isinstance(value, list):
+                raise ValueError(
```

Comment:
Shouldn't this be a TypeError ? :-D

Reply:
ah you caught me copying and pasting from the 2d mesh @property and forgot to implement the pullrequest review suggestions

Reply:
I fixed a few more of these in https://github.com/ukaea/paramak/commit/f9f7667838209ff043c2a6e0c618774e12faed3a